contracts/Positions/PositionalMarket.sol

```diff
@@ -103,12 +103,12 @@ contract BinaryOptionMarket is OwnedWithInit, IBinaryOptionMarket {
         initialMint = _parameters.deposit;
 
         // Instantiate the options themselves
-        options.long = BinaryOption(_parameters.long);
-        options.short = BinaryOption(_parameters.short);
+        options.long = Position(_parameters.long);
+        options.short = Position(_parameters.short);
         // abi.encodePacked("sLONG: ", _oracleKey)
         // consider naming the option: sLongBTC>50@2021.12.31
-        options.long.initialize("Binary Option Long", "sLONG", _parameters.limitOrderProvider, _parameters.thalesAMM);
-        options.short.initialize("Binary Option Short", "sSHORT", _parameters.limitOrderProvider, _parameters.thalesAMM);
+        options.long.initialize("Position Up", "UP", _parameters.limitOrderProvider, _parameters.thalesAMM);
+        options.short.initialize("Position Down", "DOWN", _parameters.limitOrderProvider, _parameters.thalesAMM);
```

Comment:
thank you for the review, I was pointing to the description of the long/short only on initialisation in BinaryOptionMarket since only here we have sShort and sLong, wasn't sure if we are changing the whole long/short thing, but I guess it makes sense.